packages/calypso-build/webpack/minify.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External Dependencies
+ */
+const TerserPlugin = require( 'terser-webpack-plugin' );
+
+const terserDefaults = {
+	ecma: 5,
+	safari10: true,
+	mangle: true,
+};
+
+/**
+ * Returns an array containing a Terser plugin object to be used in Webpack minification.
+ *
+ * @see https://github.com/webpack-contrib/terser-webpack-plugin for complete descriptions of options.
+ *
+ * @param {Object}           _               Options
+ * @param {(Boolean|string)} _.cache         Enable file caching
+ * @param {(Boolean|number)} _.parallel      Use multi-process parallel running to improve the build speed
+ * @param {Boolean}          _.sourceMap     Use source maps to map error message locations to modules
+ * @param {Object}           _.terserOptions Terser minify options
+ * @returns {Object[]}                       Terser plugin object to be used in Webpack minification.
+ */
+module.exports = ( { cache = true, parallel = false, sourceMap = false, terserOptions = {} } ) => [
```

Comment:
I'd like to avoid introducing our own defaults here. The plugin will have its own defaults, which we might expect to use, and what seem like sensible defaults here may conflict with expectations.

Reply:
nice. good call!